frontend/src/views/auth/NavigationAuth.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+  <v-app-bar
+    v-if="$vuetify.breakpoint.mdAndUp"
+    app
+    clipped-left
+    color="blue-grey darken-4"
+    dark
+  >
+    <logo text />
+    <v-spacer />
+    <v-toolbar-items>
+      <v-btn :href="newsLink" target="_blank" text>
+        <v-icon left>mdi-script-text-outline</v-icon>
+        {{ $tc('global.navigation.news') }}
+      </v-btn>
+    </v-toolbar-items>
```

Comment:
you need to hide the button if the link is not configured (if we want to enable a not configured value)